### ISSUE TITLE
Do not detect some windows resize correctly on Linux

### ISSUE THREAD

**Issue by sunix** [manusa/electronim#78]
Resizing works when you drag the corner of the window: the content is correctly updated with the new resize. But when the resize is done by turning full screen a window, it is not detected.

**Comment by KirilOkun** [comment]
I'm seeing strange resize behavior on linux as well.  Frequently i see the window is full screen but the content is taking up a fraction of the space with the rest filled with just white background.   The only way to fix it is to restart the app.

**Comment by manusa** [comment]
I already have a fix for this, but I'm currently waiting on another issue to be fixed before releasing.

> The only way to fix it is to restart the app.

By activating a different tab, the resize should happen.
